e2e-tests/cube-projects/delete.hydra

```diff
@@ -38,7 +38,7 @@ With Class api:EntryPoint {
 With Class cc:CubeProject {
     Expect Link cc:csvMapping {
         Expect Property cc:csvSourceCollection {
-            Expect Operation cc:UploadCSVAction {
+            Expect Operation schema:CreateAction {
```

Comment:
I notice that the body of these requests are not what we now submit but the tests pass.

We should add a `hydra:expects` to that operation and a simple shape to describe the file description.

Reply:
I added a shape to validate the input instead.